datasetinsights/configs/vgg_slam.yaml

```diff
@@ -0,0 +1,31 @@
+estimator: VGGSlam
+backbone: VGG
+task: depth_estimation
+train:
+  dataset: SingleCube
+  batch_size: 50
+  accumulation_steps: 8
+  crop_size: 769
+  epochs: 20
+val:
+  dataset: synthetic
+  batch_size: 10
+test:
+  dataset: synthetic
+  batch_size: 4
+metric: AMSE
+optimizer:
+  name: Adam
+  args:
+    lr: 0.0001
+    beta_1: 0.9
+    beta_2: 0.999
```

Comment:
We may want to set beta_1 and beta_2 as default values. Remove these two variables from all yaml files.